src/app/app.component.html

```diff
@@ -291,11 +291,11 @@ <h1 *ngIf="!fatalMessageDetail" class="u-text-body u-text-center">
 				</div>
 
 				<div class="tw-flex tw-flex-col tw-min-w-fit">
-					<a class="faq" [routerLink]="['/public/faq']">FAQ</a>
+					<a class="tw-text-[14px] tw-leading-[16.8px] tw-font-semibold  md:tw-text-[20px] md:tw-font-medium md:tw-leading-[26px] tw-text-purple" [routerLink]="['/public/faq']">FAQ</a>
 					<div>
 						<a class="impressum" [href]="theme.termsOfServiceLink">{{ 'Start.imprint' | translate }}</a>
```

Comment:
Can you remove the "impressum" classes after deleting them and also use the same style for the imprint as for ```FAQ``` and ```privacy Policy```?